cdk/package.json

```diff
@@ -27,13 +27,13 @@
     "prettier": "^2.6.2",
     "ts-jest": "^27.1.4",
     "ts-node": "^10.2.0",
-    "typescript": "~4.4.3"
-  },
-  "dependencies": {
+    "typescript": "~4.4.3",
     "aws-cdk": "2.23.0",
     "aws-cdk-lib": "2.23.0",
     "constructs": "10.1.5",
-    "@guardian/cdk": "43.2.0",
+    "@guardian/cdk": "43.2.0"
+  },
+  "dependencies": {
     "source-map-support": "^0.5.16"
```

Comment:
Could this dependency be moved to dev as well?

Reply:
Not sure - I will take a look and raise in a seperate PR if it can be